download_s3.go

```diff
@@ -0,0 +1,104 @@
+// Helper methods for downloading a file over HTTP
+
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"regexp"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/feature/s3/manager"
+	"github.com/aws/aws-sdk-go-v2/service/s3"
+	"github.com/sirupsen/logrus"
+)
+
+func s3Downloader(context context.Context) (*manager.Downloader, error) {
```

Comment:
why do all this work and not use something like https://github.com/hashicorp/go-getter ? its actually what we used for HTTP before we needed to support MD5s

Reply:
Yep! That library would have been helpful to know beforehand. If you prefer I can spend the additional time to use that instead.